lib/questions.rb

```diff
@@ -63,33 +84,47 @@ def separate_array_into_even_and_odd_numbers(array)
 # e.g. 'bob'. So in the array ['bob', 'radar', 'eat'], there
 # are 2 palindromes (bob and radar), so the method should return 2
 def number_of_elements_that_are_palindromes(array)
+  new = array.push(array.map{|array| array.reverse}).flatten
```

Comment:
Shadowing outer local variable - `array`.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Pass `&:reverse` as an argument to `map` instead of a block.